src/test/groovy/com/jvm_bloggers/core/newsletter_issues/domain/LastFiveIssuesQuerySpec.groovy

```diff
@@ -0,0 +1,46 @@
+package com.jvm_bloggers.core.newsletter_issues.domain
+
+import com.jvm_bloggers.core.data_fetching.blog_posts.domain.BlogPost
+import com.jvm_bloggers.core.data_fetching.blogs.domain.Blog
+
+import java.time.LocalDate
+import java.util.stream.Collectors
+
+class LastFiveIssuesQuerySpec extends NewsletterIssueRepositorySpecBase {
```

Comment:
I think it's better to have all test setup in one class as it does not required reader to check what is happening in the parent very often.